students/JamesTraub/Lesson02/grid_printer2.py

```diff
@@ -0,0 +1,19 @@
+# chose to divide n by 2 to give columns an even width
+
+def print_grid(n,s):
+    pls = '+'
+    mns = '-'
+    pl = '|'
+    spc = ' '
+    dventry = s
+    grdsz = (dventry * mns)
+    spcsz = (dventry * spc)
+    plsln = (pls + grdsz + pls + grdsz + pls)
+    plline = (pl + spcsz + pl + spcsz + pl)
+    for b in range(n):
```

Comment:
it is a common convention in python to use `_` var name for throw-away vars in loops. it clearly indicates that this var is never used and pretty much ignore it. Example:
```
for _ in range(n)
```